### ISSUE TITLE
Accessibility: fix text contrast

### ISSUE THREAD

**Issue by EvgenyOrekhov** [solids/npmsearch#58]
Most of the text fail the WCAG standard (and probably other accessibility standards).
It's hard to read it even with normal vision.

Examples:

- package description color: http://webaim.org/resources/contrastchecker/?fcolor=89a19d&bcolor=ffffff
- package keywords color: http://webaim.org/resources/contrastchecker/?fcolor=d8d8d8&bcolor=ffffff
- green badge: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=5ac878
- orange badge: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=ffb726

Also, it's important to note that these fonts on npmsearch.com is smaller and thinner than the "normal" font on webaim.org which makes npmsearch.com even less accessible.

**Comment by EvgenyOrekhov** [comment]
Closing due to lack of activity.
